Countries/India/legal-cookies.html

```diff
@@ -40,14 +40,18 @@ <h2>What are your choices regarding cookies</h2>
 <p>Please note, however, that if you delete cookies or refuse to accept them, you might not be able to use all of the features we offer, you may not be able to store your preferences, and some of our pages might not display properly.</p>
 
 <ul>
-	<li>For the Chrome web browser, please visit <a href="https://support.google.com/accounts/answer/32050" target="_blank">this page from Google</a>.</li>
-	<li>For the Internet Explorer web browser, please visit <a href="http://support.microsoft.com/kb/278835" target="_blank">this page from Microsoft</a>.</li>
-	<li>For the Firefox web browser, please visit this page from  <a href="https://support.mozilla.org/en-US/kb/delete-cookies-remove-info-websites-
+  <li>For the Chrome web browser, please visit <a href="https://support.google.com/accounts/answer/32050" target="_blank">this page from Google</a>.</li>
+  <li>For the Internet Explorer web browser, please visit <a href="http://support.microsoft.com/kb/278835" target="_blank">this page from Microsoft</a>.</li>
+  <li>For the Firefox web browser, please visit this page from  <a href="https://support.mozilla.org/en-US/kb/delete-cookies-remove-info-websites-
     stored" target="_blank">this page from Mozilla</a>.</li>
-	<li>For the Safari web browser, please visit <a href="https://support.apple.</li>com/kb/PH21411?locale=en_US" target="_blank">this page from Apple</a>.</li>
-	<li>For any other web browser, please visit your web browser's official web pages.</li>
+  <li>For the Safari web browser, please visit <a href="https://support.apple.</li>com/kb/PH21411?locale=en_US" target="_blank">this page from Apple</a>.</li>
+  <li>For any other web browser, please visit your web browser's official web pages.</li>
 </ul>
 
 <h2>Where can you find more information about cookies</h2>
 
+<<<<<<< HEAD
 <p>You can learn more about cookies on <a href="http://www.allaboutcookies.org/" target="_blank">AllAboutCookies</a> and <a href="http://www.networkadvertising.org/" target="_blank">Network Advertising Initiative</a>.</p>
```

Comment:
Git conflict message is added over here.

Reply:
Tnx!